### DISCUSSION TITLE
Question - The footer on the 404 page is completely different from the regular

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#328] [Q&A]
While the 404 template can be edited via WP’s website customizer → templates, there doesn’t seem to be a way to use the same footer as in the regular template. Ideally, all templates should default to the same footer.

Is there a reasons for this?

![image](https://github.com/user-attachments/assets/e894d621-8bad-4256-99a4-a1c4592624ae)

Question from @amnesty-service

**Comment by jaymcp** [comment]
The 404 template is one of the new FSE templates, as per https://github.com/amnestywebsite/amnesty-wp-theme/issues/2790. For this template, the footer was completely rebuilt using core blocks in https://github.com/amnestywebsite/amnesty-wp-theme/issues/2789

Most of the rest of the site still uses PHP templates, with our old footer implementation. These will be migrated in https://github.com/amnestywebsite/humanity-theme/issues/246

It looks like there's one or two things that need tweaking with the FSE footer, which we should open issues for:
- bug: font family for the titles needs fixing (uses an old CSS var instead of the new one)
- enhancement: spacing of the social links could do with some adjusting

Beyond those two issues, I see no significant differences between the old style footer and the FSE one. Is there anything in particular you are concerned about?

**Comment by paulHasselkuss** [comment]
I am responsible for the website at @amnesty-service and originally raised the issue. Thanks for your fast answer that resolves my initial confusion. Once everything is migrated to FSE, I believe the implementation will be both clear and usable.

However, there is one minor issue remaining: the FSW footer is not sticky. In the screenshot above, I would have expected the footer to stick to the bottom of the window. As you can see below, the old (non FSW) footer is sticky

<img width="1242" alt="Bildschirmfoto 2024-08-04 um 18 10 22" src="https://github.com/user-attachments/assets/b419b9ad-02d6-4f30-b858-0827bb44b3c8">
